MinPlatformPkg/Test/Library/TestPointCheckLibNull/TestPointCheckLibNull.c

```diff
@@ -636,7 +636,27 @@ TestPointSmmEndOfDxeSmrrFunctional (
 }
 
 /**
-  This service verifies the validity of the SMM memory atttribute table at SMM Ready To Lock.
+  This service verifies SMRR configuration at the End of DXE.
+
+  Test subject: SMRR.
+  Test overview: Verify SMRR is aligned and SMRR matches SMRAM_INFO.
+  Reporting mechanism: Set ADAPTER_INFO_PLATFORM_TEST_POINT_STRUCT.
+                       Dumps SMRR and SMRAM_INFO.
+
+  @retval EFI_SUCCESS         The test point check was performed successfully.
+  @retval EFI_UNSUPPORTED     The test point check is not supported on this platform.
+**/
+EFI_STATUS
+EFIAPI
+TestPointStandaloneMmEndOfDxeSmrrFunctional (
```

Comment:
I guess if we do not use these functions externally, these functions do not need to be implemented here?

Reply:
They were removed.